Task 2/Task_2_1_2/Task_2_1_2/Figures/Shape.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_2_1_2
+{
+    public abstract class Shape
+    {
+        
```

Comment:
```suggestion
        public abstract override string ToString();
```
Я бы сделал вот так, чтобы обязать всех наследников переопределять ToString()